task/platform_adapter.go

```diff
@@ -17,19 +17,19 @@ type RunController interface {
 
 // PlatformAdapter wraps a task.Store into the platform.TaskService interface.
 func PlatformAdapter(s backend.Store, r backend.LogReader, rc RunController) platform.TaskService {
-	return pAdapter{s: s, r: r, rc: rc}
+	return pAdapter{S: s, R: r, RC: rc}
 }
 
 type pAdapter struct {
-	s  backend.Store
-	rc RunController
-	r  backend.LogReader
+	S  backend.Store
```

Comment:
These fields shouldn't be exported. If you need to hold a reference to them, keep the reference from before the call to `PlatformAdapter`.